### PR TITLE
Pass more variable in useTable Antd

### LINKED ISSUE #3246: [FEAT] Pass more variable in useTable Antd
### Is your feature request related to a problem? Please describe.

I want to use these methods and variable.
```
current
setCurrent
pageSize
setPageSize
pageCount
```
These variables are used to manipulate the pagination system

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Describe the thing to improve

These methods should be passed to response of `useTable (AntD)` the same way `setFilters` and `setSorter` has been passed

### PR DESCRIPTION
**Pull request by rajaomariajaona**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

I just passed pagination variable into useTable result and I change types definitions

### Test plan (required)

I run the tests all passed

### Closing issues

Put `closes #3246` in your comment to auto-close the issue that your PR fixes (if such).

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by rajaomariajaona** [comment]
closes #3246

**Review by omeraplak — APPROVED** [review]
LGTM! ⚡️

**Comment by rajaomariajaona** [comment]
I don't think that error comes for me? Is it? 
how to fix it?

**Comment by omeraplak** [comment]
> I don't think that error comes from me? Is it? how to fix it?

No, it's about our CI setup. You can ignore it ⚡️

**Review by yildirayunlu — APPROVED** [review]

**Review by salihozdemir — APPROVED** [review]

**Comment by alicanerdurmaz** [comment]
Hi @rajaomariajaona, can you please add a [changeset](https://refine.dev/docs/contributing/#changeset)?

**Comment by rajaomariajaona** [comment]
> Hi @rajaomariajaona, can you please add a [changeset](https://refine.dev/docs/contributing/#changeset)?

Ok ok I'm on it

**Review by alicanerdurmaz — APPROVED** [review]

File: .changeset/pink-cats-hope.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@pankod/refine-antd": minor
+---
+
+Pass pagination variables in UseTable hooks
```
Comment (aliemir) [inline .changeset/pink-cats-hope.md:5]:
```suggestion
Add ability to pass pagination values in `useTable` hook. (Resolves #3246)
- `current`
- `setCurrent`
- `pageSize`
- `setPageSize`
- `pageCount`
```

Comment (aliemir) [reply .changeset/pink-cats-hope.md:5]: Just left a small update on changeset to be more descriptive 😅 What do you think @alicanerdurmaz @rajaomariajaona?

**Comment by omeraplak** [comment]
Thank you @rajaomariajaona 🚀

**Pull request merged**